README.md

```diff
@@ -1,32 +1,89 @@
-# Data Refinery
+# Data Refinery [![Build Status](https://circleci.com/gh/data-refinery/data-refinery/tree/dev.svg?&style=shield)](https://circleci.com/gh/data-refinery/data-refinery/)
 
 A project to download, process, aggregate, and serve bioinformatic data now
 supported by Alex's Lemonade Stand Foundation, with some initial development
 supported by the Gordon and Betty Moore Foundation via GBMF 4552 to Casey
 Greene.
 
-## Getting Started
+## Development
+
+### Git Workflow
+
+`data-refinery` uses a [feature branch](http://nvie.com/posts/a-successful-git-branching-model/ based workflow. New features should be
+developed on new feature branches, and pull requests should be sent to
+the `-dev` branch for code review. Merges into `-master` happen at the end
+of sprints, and tags in `master` correspond to production releases.
+
+### Running Locally
 
-Note: The following steps assume you have already installed PostgreSQL (>=9.4)
-and Python (>=3.5) on Ubuntu (Tested with 16.04. It should be possible to use
-other versions or even a Mac though).
+_Note: The following steps assume you have already installed PostgreSQL (>=9.4)
+and Python (>=3.5) on Ubuntu 16.04 and higher, as well as OSX. OSX users will
+need [Docker for Mac](https://www.docker.com/docker-mac) installed, as well
+as Homebrew, for to `brew install iproute2mac git-crypt git-lfs`._
```

Comment:
Having postgres installed should no longer be required since it's running in a Docker container right?